frontend/src/components/cluster/Chooser.tsx

```diff
@@ -101,6 +102,11 @@ const useStyles = makeStyles(theme => ({
     alignItems: 'center',
     display: 'flex',
   },
+  h1: {
+    fontSize: '1.25rem',
+    fontWeight: 500,
+    lineHeight: 1.6,
+  },
```

Comment:
Ditto.

Reply:
An alternative would be to use a wrapper component so we don't have to call typography + style all the time.

I'm curious to learn more about why the h1 is needed instead of h2 since I found some examples that use h2 like
https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

Reply:
I guess that example is assuming the document also has a h1 tag in the document. For ours, the dialog component sets aria-hidden=true on the other parts of the doc, leaving it with no h1 header. Only the dialog is visible in the document at this point to readers, and so we need a h1 tag for the dialog title.

Reply:
> An alternative would be to use a wrapper component so we don't have to call typography + style all the time.

Cool, I'll do that wrapper. It will be easier to delete later.

Reply:
Yes, that's the idea. This makes things easier to fix now and also easier to replace when we move to a newer MUI. Thanks.